app/javascript/components/user-activity-counts.vue

```diff
@@ -1,44 +1,33 @@
 <template lang="pug">
 .card-counts.mt-3(v-if='user.student_or_trainee')
   dl.card-counts__items
-    .card-counts__item
-      .card-counts__item-inner
-        dt.card-counts__item-label
-          | 日報
-        dd.card-counts__item-value(
-          :class='user.report_count == 0 ? "is-empty" : ""')
-          | {{ user.report_count }}
-    .card-counts__item
-      .card-counts__item-inner
-        dt.card-counts__item-label
-          | 提出物
-        dd.card-counts__item-value(
-          :class='user.product_count == 0 ? "is-empty" : ""')
-          | {{ user.product_count }}
-    .card-counts__item
-      .card-counts__item-inner
-        dt.card-counts__item-label
-          | コメント
-        dd.card-counts__item-value(
-          :class='user.comment_count == 0 ? "is-empty" : ""')
-          | {{ user.comment_count }}
-    .card-counts__item
-      .card-counts__item-inner
-        dt.card-counts__item-label
-          | 質問
-        dd.card-counts__item-value(
-          :class='user.question_count == 0 ? "is-empty" : ""')
-          | {{ user.question_count }}
-    .card-counts__item
-      .card-counts__item-inner
-        dt.card-counts__item-label
-          | 回答
-        dd.card-counts__item-value(
-          :class='user.answer_count == 0 ? "is-empty" : ""')
-          | {{ user.answer_count }}
+    user-activity-count(
+      :activity-name='`日報`',
```

Comment:
`activity-name`に渡す値は静的なので、コロンはなくてもいいかなと思いましたがどうでしょうか。
[props | Vue.js](https://ja.vuejs.org/guide/components/props#static-vs-dynamic-props)

また、各文字列がバッククォートで囲まれていますので、こちらの削除もお願いいたします。
```suggestion
      activity-name='日報',
```

Reply:
失礼しました😅
これは調査・確認不足でした💦